### DISCUSSION TITLE
Community member shoutout - benz0li

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#5365] [Announcements]
We want to take a second to give @benz0li a huge shout-out for their amazing support with the code-server project! They've written [fantastic bug reports](https://github.com/coder/code-server/issues/5031), [helped other issues solve problems](https://github.com/coder/code-server/issues/5334#issuecomment-1191189358), given us [honest feedback](https://github.com/coder/code-server/discussions/5326) and done so much to help this project and community. We can't tell you how much we appreciate it. 

From everyone on the code-server team & Coder, thank you! 🙏🏼 🙌🏼

**Comment by benz0li** [comment]
Thank you!

This is simply a _quid pro quo_ for all the work that devs @coder have put into code-server.

https://github.com/coder/code-server/pull/503 wasn't me, though 😉.

**Reply by jsjoeio** [reply]
😅 wrong link! Darn you clipboard history. Fixed!
